--- ServerRequestFactory.php.orig	2023-06-10 11:22:43.919751867 +0000
+++ ServerRequestFactory.php	2023-06-10 11:23:37.851298893 +0000
@@ -225,7 +225,7 @@
      */
     public static function createUri(array $server = []): UriInterface
     {
-        $server += $_SERVER;
+        $server = __cakefuzzer_array_merge($server, $_SERVER); // CakeFuzzerInstrumentation:patched
         $server = normalizeServer($server);
         $headers = marshalHeadersFromSapi($server);
 
