--- /var/www/cerebrate/vendor/laminas/laminas-diactoros/src/functions/marshal_headers_from_sapi.php.orig	2023-03-16 14:34:49.293483827 +0000
+++ /var/www/cerebrate/vendor/laminas/laminas-diactoros/src/functions/marshal_headers_from_sapi.php	2023-03-16 14:35:25.857375657 +0000
@@ -15,7 +15,7 @@
  * @param array $server Values obtained from the SAPI (generally `$_SERVER`).
  * @return array Header/value pairs
  */
-function marshalHeadersFromSapi(array $server): array
+function marshalHeadersFromSapi($server): array // CakeFuzzerInstrumentation
 {
     $contentHeaderLookup = isset($server['LAMINAS_DIACTOROS_STRICT_CONTENT_HEADER_LOOKUP'])
         ? static function (string $key): bool {
