--- /var/www/cerebrate/vendor/laminas/laminas-diactoros/src/functions/marshal_uri_from_sapi.php.orig	2023-03-16 14:37:11.021064430 +0000
+++ /var/www/cerebrate/vendor/laminas/laminas-diactoros/src/functions/marshal_uri_from_sapi.php	2023-03-16 15:26:00.596545921 +0000
@@ -1,5 +1,5 @@
 <?php
-
+ // CakeFuzzerInstrumentation:patched
 declare(strict_types=1);
 
 namespace Laminas\Diactoros;
@@ -179,9 +179,9 @@
 
         return 'on' === strtolower($https);
     };
-    if (array_key_exists('HTTPS', $server)) {
+    if (__cakefuzzer_array_key_exists('HTTPS', $server)) {
         $https = $marshalHttpsValue($server['HTTPS']);
-    } elseif (array_key_exists('https', $server)) {
+    } elseif (__cakefuzzer_array_key_exists('https', $server)) {
         $https = $marshalHttpsValue($server['https']);
     } else {
         $https = false;
