--- /var/www/cerebrate/vendor/laminas/laminas-diactoros/src/functions/normalize_server.php.orig	2023-03-16 13:50:24.005358506 +0000
+++ /var/www/cerebrate/vendor/laminas/laminas-diactoros/src/functions/normalize_server.php	2023-03-16 15:25:21.578191696 +0000
@@ -18,7 +18,7 @@
  *     `apache_request_headers` under the Apache mod_php.
  * @return array Either $server verbatim, or with an added HTTP_AUTHORIZATION header.
  */
-function normalizeServer(array $server, ?callable $apacheRequestHeaderCallback = null): array
+function normalizeServer($server, ?callable $apacheRequestHeaderCallback = null) // CakeFuzzerInstrumentation:patched
 {
     if (null === $apacheRequestHeaderCallback && is_callable('apache_request_headers')) {
         $apacheRequestHeaderCallback = 'apache_request_headers';
